buildconfig/CMake/Eigen.cmake

```diff
@@ -0,0 +1,25 @@
+include(ExternalProject)
+
+# Use version 3.2.10 of Eigen
+set(eigen_version "3.2.10")
```

Comment:
Owen suggest to add a comment here why you went back to the old version (compiler warnings in the latest builds).